### PR TITLE
AIRAVATA-3046 and updating out of date thrift stubs with changes

### PR DESCRIPTION
**Pull request by shivamriky**
Summary of changes:

- Changing directory name from **job** to **Job** in thrift stub as Linux is case-sensitive.

- Updating the airavata-client-sdk  with latest cpp stubs

- Adding deprecated comment to the addUserProfile methods as it is not used in the Django portal

- Fixing a typo in POM file. Varialbes use curly braces instead of round brackets. 

- Updating some of other pending changes in the thrift stubs .

### CODE REVIEW

File: thrift-interface-descriptions/service-cpis/profile-service/profile-user/profile-user-cpi.thrift [hunk 1]
```diff
@@ -47,7 +47,11 @@ service UserProfileService {
   string initializeUserProfile (1: required security_model.AuthzToken authzToken)
                             throws (1: profile_user_cpi_errors.UserProfileServiceException upe,
                                     2: airavata_errors.AuthorizationException ae);
-
+  /**
+   * This method is deprecated and will be removed in future versions of our Django portal. Please use
```
Comment (machristie) [inline thrift-interface-descriptions/service-cpis/profile-service/profile-user/profile-user-cpi.thrift]: I would say "in future versions of Airavata".

Comment (shivamriky) [reply thrift-interface-descriptions/service-cpis/profile-service/profile-user/profile-user-cpi.thrift]: Sure, @machristie .
I will update it.

Comment (shivamriky) [reply thrift-interface-descriptions/service-cpis/profile-service/profile-user/profile-user-cpi.thrift]: I have made the requested change.

**Comment by machristie** [comment]
Hi @shivamriky , looks like there are merge conflicts now. Can you merge in develop and regenerate the stubs?

**Comment by shivamriky** [comment]
Okay. I will check it.

**Comment by shivamriky** [comment]
> Hi @shivamriky , looks like there are merge conflicts now. Can you merge in develop and regenerate the stubs?

@machristie I have rebased my commits on the top of latest git commits.

**Pull request merged**